src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_api.py

```diff
@@ -56,6 +56,8 @@ def set_ide_os_and_breakpoints_by(self, py_db, seq, ide_os, breakpoints_by):
 
         pydevd_file_utils.set_ide_os(ide_os)
 
+        if seq == 0:
```

Comment:
I think you shouldn't do this here... if customization is needed it should be done in `NetCommandFactoryJson` (where it can always return `NULL_NET_COMMAND`).